mutters-core/src/main/java/com/rabidgremlin/mutters/core/Intent.java

```diff
@@ -1,17 +1,21 @@
+/* Licensed under Apache-2.0 */
 package com.rabidgremlin.mutters.core;
 
 import java.util.Collection;
 import java.util.Collections;
 
 /**
- * This class represents an intent. Each intent has a unique name and zero or more Slots that are used to extract
- * entities out of a user's input.
+ * This class represents an intent. Each intent has a unique name and zero or
+ * more Slots that are used to extract entities out of a user's input.
  * 
  * @author rabidgremlin
  *
  */
 public class Intent
 {
+  /** Intent used to indicate no match. */
+  public final static Intent NONE = new Intent("NONE");
```

Comment:
maybe make it private and use a static method to retrieve
like how ```Optional.empty()``` works

Reply:
mmmm, Will investigate.

Reply:
Implemented.